pipeline/article_scraper/AmisScraper/settings.py

```diff
@@ -15,12 +15,13 @@
 NEWSPIDER_MODULE = 'AmisScraper.spiders'
 
 
-#LOG_STDOUT = True
-#LOG_FILE = 'scrapy_output.log'
+LOG_STDOUT = True
+LOG_FILE = 'scrapy_output.log'
 
 
-# Crawl responsibly by identifying yourself (and your website) on the user-agent
-#USER_AGENT = 'amisSpider (+http://www.yourdomain.com)'
+# Crawl responsibly by identifying yourself (and your website) on the
```

Comment:
we could remove this patronizing bullshit.
It's a leftover of the autogenerated code...